### DISCUSSION TITLE
一周前邮箱提交了退款申请，能不能有人搭理一下？

### DISCUSSION THREAD

**Discussion by lovechenyulu** [getlantern/lantern-forum-cn#412] [Q&A]
3.27购买了专业版，之后一直不能用，4.1提交了退款申请的邮件，现在4.8了，一周过去了，还是没有人跟进。

**Comment by MaxMKMax** [comment]
@lovechenyulu 您好. 非常抱歉. 估计是支付团队那里收到的邮件太多了. 请您提供我账号邮箱和单据. 我来帮您催一下!

**Reply by lovechenyulu** [reply]
账号邮箱： z h a n g y u _  1 8 9 5 @ 1 6 3 . c o m

**Reply by lovechenyulu** [reply]
![Uploading 账单.png…]()

**Reply by lovechenyulu** [reply]
又过去一周了，为什么还是没有反馈呢？

**Reply by MaxMKMax** [reply]
@lovechenyulu 您好! 付款团队今天会帮您弄退款了. 请您耐心等待.

**Reply by MaxMKMax** [reply]
@lovechenyulu 您好. 支付团队需要您重新发付款截图一下. 这截图看不了. 麻烦您了, 谢谢!

**Reply by lovechenyulu** [reply]
付款截图用邮件回复了，请问收到了吗？@

**Reply by MaxMKMax** [reply]
@lovechenyulu 您好. 我 3 周之前有帮您提交了. 但我在邮件或都看不到您的付款截图. 完全看不到也下载不了. 
我刚才帮您问付款团队了, 看能否没有截图可以退款. 但您可以的话再试多几次截图这里给我.

![image](https://github.com/getlantern/lantern-forum-cn/assets/155475142/1bd67253-8c58-4e28-80e6-19ba6583de76)

**Comment by lovechenyulu** [comment]
付款截图请查看附件图片













在 2024-04-19 01:14:14，"Mike K" ***@***.***> 写道：

@lovechenyulu 您好. 支付团队需要您重新发付款截图一下. 这截图看不了. 麻烦您了, 谢谢!

**Comment by MaxMKMax** [comment]
@lovechenyulu 还是您把截图放到Google doc还是Google drive可以吗? 不然您发给我的全被拦截下来了. 完全看不到. 
不好意思麻烦您了. 谢谢您的帮忙. 您这边的支付单据弄好了就能马上给您弄退款!

**Comment by lovechenyulu** [comment]
截图放到google drive里了，请下载查看




https://drive.google.com/file/d/11irEVligtzG5AEBWaTC0WB-YySgoBCn-/view?usp=drive_link
















在 2024-05-09 01:21:34，"Mike K" ***@***.***> 写道：

@lovechenyulu 还是您把截图放到Google doc还是Google drive可以吗? 不然您发给我的全被拦截下来了. 完全看不到

**Reply by MaxMKMax** [reply]
![image](https://github.com/getlantern/lantern-forum-cn/assets/155475142/3fcf35d9-5421-4f4d-8093-25d486bc8f5b)

@lovechenyulu 您好. 谢谢您放在drive. 请您给我权限或者设定他为全面开放访问.
mike@getlantern.org

**Comment by lovechenyulu** [comment]
已经开放权限了，请查看
















At 2024-05-15 00:58:17, "Mike K" ***@***.***> wrote:

image.png (view on web)

@lovechenyulu 您好. 谢谢您放在drive. 请您给我权限或者设定他为全面开放访问.
***@***.***

**Comment by lovechenyulu** [comment]
已经开放了全部查看的权限，请问能看到吗？

https://drive.google.com/file/d/11irEVligtzG5AEBWaTC0WB-YySgoBCn-/view?usp=drive_link













At 2024-05-15 00:58:17, "Mike K" ***@***.***> wrote:

image.png (view on web)

@lovechenyulu 您好. 谢谢您放在drive. 请您给我权限或者设定他为全面开放访问.
***@***.***

**Reply by MaxMKMax** [reply]
@lovechenyulu 可以看到. 我已经报您提交了.

**Comment by lovechenyulu** [comment]
你好啊，目前还没有收到退款，请问进度如何了，大概什么时候能收到呢？
















在 2024-05-21 01:22:01，"Mike K" ***@***.***> 写道：

@lovechenyulu 可以看到. 我已经报您提交了.

**Comment by lovechenyulu** [comment]
两周了，还没有收到退款。大概什么时候能退呢？
















在 2024-05-21 01:22:01，"Mike K" ***@***.***> 写道：

@lovechenyulu 可以看到. 我已经报您提交了.

**Reply by MaxMKMax** [reply]
@lovechenyulu 
您好. 我们的负责人在假期. 照道理应该早已经退款了, 等他回来我再帮您检查.  不好意思啊.

**Comment by lovechenyulu** [comment]
这是我的账号邮箱：




 ***@***.***




在 2024-06-05 23:12:01，"Mike K" ***@***.***> 写道：

@lovechenyulu
您好. 我们的负责人在假期. 早道理应该已经退款了, 等他回来我再帮您检查. 不好意思啊. 您方便再给我一次您的账号邮箱吗?

**Comment by lovechenyulu** [comment]
我6月17号给你发送了账号邮箱，你收到了吗？ ***@***.***。

请问什么时候能完成退款呢？










在 2024-06-05 23:12:01，"Mike K" ***@***.***> 写道：

@lovechenyulu
您好. 我们的负责人在假期. 早道理应该已经退款了, 等他回来我再帮您检查. 不好意思啊. 您方便再给我一次您的账号邮箱吗?

**Reply by MaxMKMax** [reply]
@lovechenyulu 
您好. 您今天再发个邮件给我. 我帮您问看. 给我支付单据和账号邮箱. 你这里的账号邮箱我看不到. 被屏蔽了.
mike@getlantern.org

**Comment by MaxMKMax** [comment]
@lovechenyulu 您好. 或者您用这个新的方式重新提交问题吧.

为了保护您的隐私, 请您直接在这个链接上提交您的资料就好了. 支付单据和邮箱. 
后续您可以在GitHub这个帖子和我跟进如果您的问题还没解决. 谢谢您!
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ

**Comment by lovechenyulu** [comment]
你给我的链接里要我提供完成购买后收到的付款确认电子邮件的截图，我在付款成功后没有收到付款确认的邮件，我只有通过支付宝付款成功的界面截图。

我的邮箱是 ***@***.*** 。支付单据如下。




在 2024-06-26 17:09:29，"Mike K" ***@***.***> 写道：

@lovechenyulu 您好. 或者您用这个新的方式重新提交问题吧.

为了保护您的隐私, 请您直接在这个链接上提交您的资料就好了. 支付单据和邮箱.
后续您可以在GitHub这个帖子和我跟进如果您的问题还没解决. 谢谢您!
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ

**Reply by MaxMKMax** [reply]
@lovechenyulu 您好. 您在链接里提供支付宝证明就可以了.

**Comment by lovechenyulu** [comment]
我在连接里提交了，请你核对一下







在 2024-07-02 00:02:01，"Mike K" ***@***.***> 写道：

@lovechenyulu 您好. 您在链接里提供支付宝证明就可以了.

**Comment by lovechenyulu** [comment]
又过了一周了，还是没有退款，请问有人处理吗?













在 2024-07-02 00:02:01，"Mike K" ***@***.***> 写道：

@lovechenyulu 您好. 您在链接里提供支付宝证明就可以了.

**Comment by MaxMKMax** [comment]
@lovechenyulu 您检查一下, 退款应该早就完成了. 都这么久了. 我再帮您问看. 您的账号邮箱被屏蔽了, 我这里看不到. 您隔着写邮箱给我. 这样写 m i ke@ get lante r n. or g
我帮您去追讨一下.

**Comment by lovechenyulu** [comment]
没收到退款。

我的邮箱：

z h a n g y u _ 1 8 9 5 @ 1 6 3 . c o m







At 2024-07-15 22:28:15, "Mike K" ***@***.***> wrote:

@lovechenyulu 您检查一下, 退款应该早就完成了. 都这么久了. 我再帮您问看. 您的账号邮箱被屏蔽了, 我这里看不到. 您隔着写邮箱给我. 这样写 m i ke@ get lante r n. or g
我帮您去追讨一下.

**Comment by MaxMKMax** [comment]
@lovechenyulu 好的. 已经帮您跟进了, 等待回复. 谢谢您的耐心等待.
